.g8/addressPage/app/controllers/$package$/$className$Controller.scala

```diff
@@ -72,8 +72,8 @@ class $className;format="cap"$Controller @Inject()(
               .fold(
                 formWithErrors => Future.successful(BadRequest(view(formWithErrors, lrn, mode, name, isPostalCodeRequired))),
                 value => {
-                  implicit val navigator: UserAnswersNavigator = navigatorProvider(mode)
-                  $className$Page.writeToUserAnswers(value).updateTask().writeToSession().navigate()
+                  val navigator: UserAnswersNavigator = navigatorProvider(mode)
+                  $className$Page.writeToUserAnswers(value).updateTask().writeToSession(sessionRepository).navigate()
```

Comment:
can the .navigate calls in the scaffolds be as well updated to *.navigateWith(navigator)*